elasticgraph-indexer/sig/elastic_graph/indexer/operation/upsert.rbs

```diff
@@ -15,7 +15,6 @@ module ElasticGraph
         attr_reader prepared_record: ::Hash[::String, untyped]
 
         @to_datastore_bulk: datastoreBulkPayload?
-        @metadata: ::Hash[::Symbol, untyped]?
```

Comment:
`@metadata` type declaration is duplicated. This validation is introduced in RBS 3.9 and raises an unexpected error in Steep.